crates/bevy_diagnostic/src/frame_time_diagnostics_plugin.rs

```diff
@@ -3,32 +3,47 @@ use bevy_app::prelude::*;
 use bevy_core::Time;
 use bevy_ecs::{IntoQuerySystem, Res, ResMut};
 
-/// Adds "frame time" diagnostic to an App, specifically "frame time" and "fps"
+/// Adds "frame time" diagnostic to an App, specifically "frame time", "fps" and "frame count"
 #[derive(Default)]
 pub struct FrameTimeDiagnosticsPlugin;
 
+pub struct FrameTimeDiagnosticsState {
+    frame_count: f64,
+}
+
 impl Plugin for FrameTimeDiagnosticsPlugin {
     fn build(&self, app: &mut bevy_app::AppBuilder) {
         app.add_startup_system(Self::setup_system.system())
+            .add_resource(FrameTimeDiagnosticsState { frame_count: 0.0 })
             .add_system(Self::diagnostic_system.system());
     }
 }
 
 impl FrameTimeDiagnosticsPlugin {
     pub const FPS: DiagnosticId = DiagnosticId::from_u128(288146834822086093791974408528866909483);
+    pub const FRAME_COUNT: DiagnosticId =
+        DiagnosticId::from_u128(54021991829115352065418785002088010277);
```

Comment:
i think id prefer a fully-random uuid just so these are more distinguishable when printed. I generated one for you :)

`73441630925388532774622109383099159699`